connectivity/check/deployment.go

```diff
@@ -983,7 +983,7 @@ func (ct *ConnectivityTest) validateDeployment(ctx context.Context) error {
 		}
 	}
 
-	if ct.params.MultiCluster == "" {
+	if ct.params.MultiCluster == "" && ct.features[FeatureKPRNodePort].Enabled {
```

Comment:
Could you please explain why this test should be skipped?

This check only checks for the kube-proxy-replacement NodePort. But regular old NodePort with kube-proxy should still pass this connectivity check, and this is something we want to test as well.